### PR TITLE
Outliers

### PR DESCRIPTION
**Pull request by KFedak**
taking a break so pulling my branch changes to master. Will pick up later today or tomorrow by first pushing from master to my branch again.

### CODE REVIEW

**Comment by kbilsback** [comment]
:octocat:

**Comment by kbilsback** [comment]
yea! emojis 🌵 😸 :taco:

File: r_markdown/benzene_emissionsfactor_qa.Rmd [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  benzene <- dplyr::filter(emissions,  pol == "voc_benzene")
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}  
+  benzene <- dplyr::left_join(benzene, 
+                              dplyr::select(samples, id, fuel, stove, fuelcat, stovecat),
+                              by = "id") %>%
+             dplyr::left_join(fuel_burnt, by = "id")
```
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: both samples and fuel_burnt have a variable called "fuel" which is causing problems when you join (creates a variable fuel.x and then issues with your code does the line). I've fixed in commit...

File: r_markdown/benzene_emissionsfactor_qa.Rmd [hunk 2]
```diff
@@ -0,0 +1,170 @@
+---
```
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: I'll apply the new style template to this in a new branch after we've merged...

File: r_markdown/process_metadata.Rmd [hunk 3]
```diff
@@ -96,7 +96,7 @@ output:
   save(batch_times, file="../r_files/batch_times.Rda")
```
Comment (ngood13) [inline r_markdown/process_metadata.Rmd:96]: looks good

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(filter(toluene, fuelcat == 'wood'), aes(x= id, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  facet_grid( ~ stove, scales = 'free') +
+  ggtitle("Benzene EFs by Stove Type - Wood Stoves Only") +
```
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: Benzene or Toluene?

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+  theme(axis.text.x = element_text(angle = 90, hjust = 1))
+```
+
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(filter(toluene, fuelcat == 'charcoal'), aes(x= id, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  facet_grid( ~ stove, scales = 'free') +
+  ggtitle("Benzene EFs by Stove Type - Charcoal Stoves Only") +
```
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: Benzene or Toluene?

**Review by kbilsback — CHANGES_REQUESTED** [review]
Wait, how did those benzene and toluene files sneak in there? #roguemerge 😸

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+  facet_grid( ~ fuelcat, scales = 'free') +
+  ggtitle("Toluene EFs by Fuel Type and Stove Type") +
+  xlab("Stove type") +
+  ylab("g emitted / g fuel burned") +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45,  hjust = 0.9))
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
```
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: drop x axis labels perhaps?

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  summary(toluene$pol_fuel_mass_ratio)
+```
+
+* units: grams of pollutant emitted per gram of fuel burned **NOTE: VERIFY UNITS**
+
+# Plots
+
```
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: I'll add some ## when I do the style update for ease of navigation...

File: r_markdown/benzene_emissionsfactor_qa.Rmd [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+  benzene <- dplyr::mutate(benzene, pol_fuel_mass_ratio = mass_emitted * 1e-6 / mass_fuel)  # ug * 1e-6 / g
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  summary(benzene$pol_fuel_mass_ratio)
+```
+
+* units: grams of pollutant emitted per gram of fuel burned **NOTE: VERIFY UNITS**
+
+# Plots
```
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: see comments on toluene that are also relevant to these (and visa versa...)

**Comment by KFedak** [comment]
waaaaa I have no idea! :crying_cat_face:

File: r_markdown/benzene_emissionsfactor_qa.Rmd [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45,  hjust = 0.9))
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(benzene, aes(x= id, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  facet_wrap(fuelcat ~ stovecat) +
+  ggtitle("Benzene EFs by Stove and Fuel categories") +
```
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: we should define a manual fuel color-scale so they always appear the same regardless of subsetting (Kelsey? 👍 )

File: r_markdown/toluene_emissionsfactor_qa.Rmd [hunk 10]
```diff
@@ -0,0 +1,174 @@
+---
```
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: same as benzene (fuel conflict, fixed)

**Review by ngood13 — APPROVED** [review]
just make sure my commits are part of the merge to the master...

File: r_markdown/process_metadata.Rmd [hunk 11]
```diff
@@ -166,6 +170,8 @@ output:
           axis.title=element_text(size=40))
 ```
 
+Outlier 16B checked by KF in February 2017. There is no transcription error, but this measure is QC-flagged as bad due to an issue with the carbonyl line being clogged/having known sampling issues that impacted flow rates.
```
Comment (kbilsback) [inline r_markdown/process_metadata.Rmd:173]: Nice! Comments!

**Comment by KFedak** [comment]
I finished making my comments in the "process_metadata.Rmd" file, and am now trying to merge back to the master.... but it looks like merging is blocked until a requested change is addressed, and I am blocked from making that change?

**Comment by ngood13** [comment]
Looks @kbilsback has requested changes, not sure who they are, they sound like trouble :-)

Also, I fixed some stuff in the voc markdowns that @rowdyspecialkbnewlyreformedenforcerofrules deleted, so you might want to look back and find them if needed...

**Review by kbilsback — APPROVED** [review]

**Comment by kbilsback** [comment]
@KFedak already fixed the changes! :octocat: 👍

**Pull request merged**